docs/en/sql-reference/functions/geo/h3.md

```diff
@@ -191,31 +191,33 @@ Result:
 
 ## geoToH3 {#geotoh3}
 
-Returns [H3](#h3-index) point index `(lon, lat)` with specified resolution.
+Returns [H3](#h3-index) point index `(lat, lon)` with specified resolution.
 
 **Syntax**
 
 ```sql
-geoToH3(lon, lat, resolution)
+geoToH3(lat, lon, resolution)
 ```
 
 **Arguments**
 
-- `lon` — Longitude. [Float64](../../data-types/float.md).
 - `lat` — Latitude. [Float64](../../data-types/float.md).
+- `lon` — Longitude. [Float64](../../data-types/float.md).
 - `resolution` — Index resolution. Range: `[0, 15]`. [UInt8](../../data-types/int-uint.md).
 
 **Returned values**
 
 - Hexagon index number. [UInt64](../../data-types/int-uint.md).
 - 0 in case of error. [UInt64](../../data-types/int-uint.md).
 
+Note: In ClickHouse v25.4 or older, `geoToH3()` takes values in order `(lon, lat)`. As per ClickHouse v25.5, the input values are in order `(lat, lon)`. The previous behaviour can be restored using setting `geotoh3_lon_lat_input_order = true`.
+
 **Example**
 
 Query:
 
 ```sql
-SELECT geoToH3(37.79506683, 55.71290588, 15) AS h3Index;
+SELECT geoToH3(55.71290588, 37.79506683, 15) AS h3Index;
```

Comment:
You also need to update the result below as lon and lat are switched, the results will differ.